docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md

```diff
@@ -11,18 +11,18 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 The KnowBe4 API integration collects user events data into Sumo Logic for storage, analysis, and alerting. It ingests events data from the [Events API](https://developer.knowbe4.com/rest/userEvents#tag/Events/operation/listEvents), phishing security tests from the [Phishing Security Tests API](https://developer.knowbe4.com/rest/reporting#tag/Phishing/paths/~1v1~1phishing~1security_tests/get), and recipient results from the [Recipient Results API](https://developer.knowbe4.com/rest/reporting#tag/Phishing/paths/~1v1~1phishing~1security_tests~1%7Bpst_id%7D~1recipients/get).
 
-:::important
-Note that access to KnowBe4 APIs is limited to Platinum and Diamond customers, and the `_siemparser` is currently only available for the External Events source.
-:::
-
 ## Prerequisites
 
 Before you begin setting up your **KnowBe4** Source, which is required to connect to the KnowBe4 API, you'll need to configure your integration with the **Region** and **KnowBe4 API Token**.
 
+:::important
+KnowBe4 APIs are only limited to Platinum and Diamond customers. , and the `_siemparser` is currently only available for the External Events source.
```

Comment:
```suggestion
KnowBe4 APIs are only limited to Platinum and Diamond customers.
```

Reply:
`_siemparser` is currently only available for the External Events source. This shouldn't come here.